docs/user-guide/persistent-volumes/walkthrough.md

```diff
@@ -36,6 +36,31 @@ NAME      LABELS       CAPACITY      ACCESSMODES   STATUS      CLAIM     REASON
 pv0001    type=local   10737418240   RWO           Available 
 ```
 
+### Access Control
+Storage configured with GID will only allow writing by pods using the same GID.
+Mismatched or missing GIDs will cause `permission denied` errors. Adding a GID
+to a `PersistentVolume` via the `pv.beta.kubernetes.io/gid` annotation allows
+`kubelet` to automatically add the GID to the pod that requires it. No
+coordination between an admin and end user is required. Setting the GID
+automatically is safe because this is only done for `PersistentVolumes` which
```

Comment:
I'm not sure either and I don't think the reader needs this reassurance(?), I will remove this